identity-actor/src/actor/endpoint.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::borrow::Cow;
+use std::fmt::Display;
+use std::str::Split;
+
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::actor::Error;
+use crate::actor::Result as ActorResult;
+
+/// A path-like identifier for an actor request. As an example, `identity/create`
+/// could be an endpoint of the request "create" within the "identity" namespace.
+///
+/// Endpoints are separated by slashes and only allow alphabetic ascii characters and `_`.
```

Comment:
`slashes` in plural is a bit misleading as an endpoint may only contain a single slash.